packages/spacer/src/__gemini__/Spacer.stories-test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import Spacer from "../index";
+
+function returnMarkup(props) {
```

Comment:
_Nitpick:_ this name isn't very descriptive. something like `renderStory` would be better.